app/src/main/java/org/greenstand/android/TreeTracker/preferences/Preferences.kt

```diff
@@ -35,16 +35,16 @@ class Preferences(
 
     private fun computePath(prefKey: PrefKey): String {
         return when (prefKey) {
-            is UserPrefKey -> {
-                prefKey.path + "/${_planterInfoId}"
-            }
-            else -> {
-                prefKey.path
-            }
+            is UserPrefKey -> prefKey.path + "/${_planterInfoId}"
```

Comment:
It will be nice to have a documentation with e.g. cases where `_planterInfoId` could be used in the keypath. For instance, `UserPrefKey` is used when setting language preference. This is a scenario where the preference is persisted across Sessions correct? So If you could add documentation for `UserPrefKey` that would throw some light when someonelooks at `PrefKey` and `UserPrefKey` and wonder when to use them.

Reply:
Sorry, forgot to link to the actual docs I wrote. I can add a link later into the code to this wiki page: https://github.com/Greenstand/treetracker-android/wiki/Preferences-&-PrefKeys